core/math/random.h

```diff
@@ -0,0 +1,25 @@
+#ifndef RAND_CORE_H
+#define RAND_CORE_H
+
+#include "randombase.h"
+#include "reference.h"
+
+class Random : public Reference {
```

Comment:
Please rename to RandomNumberGenerator, so its clearer what it does. You will only use it at class initialization anyway.

Reply:
Maybe RNG is better ?

Reply:
"RNG" is already used in the code as a shortened form of "range". Please don't use "RNG". I would also like "RNG" changed to "range" at some point.

Reply:
Well, [not capitalized](https://github.com/godotengine/godot/blob/master/core/math/math_funcs.h#L227), but still could be confusing.